src/flags.zig

```diff
@@ -0,0 +1,127 @@
+const std = @import("std");
+const assert = std.debug.assert;
+
+/// Format and print an error message followed by the usage string to stderr,
+/// then exit with an exit code of 1.
```

Comment:
> followed by the usage string

Is the comment wrong or the code? I don't see a usage string.

Reply:
wrong both here and in the cli.zig which is where this was copyied from.